src/main/java/com/actiontech/dble/server/handler/SelectHandler.java

```diff
@@ -107,6 +107,9 @@ public static void handle(String stmt, ShardingService service, int offs) {
             case ServerParseSelect.ROW_COUNT:
                 SelectRowCount.response(service);
                 break;
+            case ServerParseSelect.MAX_ALLOWED_PACKET:
+                SelectMaxAllowedPacket.response(service);
```

Comment:
whether to reuse SelectMaxAllowedPacket on the management side

Reply:
can reuse